CMakeLists.txt

```diff
@@ -86,3 +104,34 @@ add_subdirectory (test)
 add_subdirectory (test/core)
 add_subdirectory (test/http)
 add_subdirectory (test/websocket)
+
+################################################################################
+# INSTALLATION                                                                 #
+################################################################################
+
+file(STRINGS include/beast/version.hpp
+             Beast_VERSION_STRING
+             REGEX "#define BEAST_VERSION_STRING \"\.+\"")
+string(REGEX MATCH "\"\.+\"" Beast_VERSION_STRING ${Beast_VERSION_STRING})
+string(REPLACE "\"" "" Beast_VERSION_STRING ${Beast_VERSION_STRING})
+set(VERSION ${Beast_VERSION_STRING})
```

Comment:
minor: I don't see any particular need for the `VERSION` variable - I believe you could just use `${Beast_VERSION_STRING}` in the `write_basic_package_version_file` invocation instead.